File: Java/UberBlack.java
import java.util.ArrayList;
import java.util.Map;

class UberBlack extends Car{
    Map<String, Map<String, Integer>> typeCarAcceted;
    ArrayList<String> seatsMaterial;
    

    public UberBlack(String license, Account driver, Map<String, Map<String, Integer>> typeCarAcceted, ArrayList<String> seatsMaterial) {
        super(license, driver);
        this.typeCarAcceted = typeCarAcceted;
        this.seatsMaterial = seatsMaterial;
    }
}

File: Python/UberBlack.py
from car import Car 

class UberBlack(Car):
    typeCarAcceted = []
    seatsMaterial = []
    
    def __init__(self,license, driver, typeCarAcceted, seatsMaterial):
        super().__init__(license, driver)
        self.typeCarAcceted = typeCarAcceted
        self.seatsMaterial = seatsMaterial
        